contractsSol5/KyberNetwork.sol

```diff
@@ -65,6 +65,27 @@ contract KyberNetwork is WithdrawableNoModifiers, Utils4, IKyberNetwork, Reentra
         emit EtherReceival(msg.sender, msg.value);
     }
 
+
```

Comment:
spacing between functions should be 1 line.

function is not in a good place.
more important functions should be on top of file.
can also see solidity coding conventions.
A good place in network would be near getNetworkData